src/api/flaskr/service/shifu/block_funcs.py

```
@@ -211,6 +211,11 @@ def save_block_list_internal(
                     )
                     app.logger.info(f"new block : {block_model.script_id}")
                     _fetch_profile_info_for_block_dto(app, block_dto)
+                    if (
+                        block_dto.profile_info != None
```

Flake8 (7.2.0)

[error] comparison to None should be 'if cond is not None:'

(E711)